package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "kiwi-ui",
   "version": "0.0.0",
-  "scripts": {},
+  "scripts": {
+    "build-icons": "babel-node icons/bin/build.js"
```

Comment:
can't we use regular node for these kinds of scripts?

Reply:
Is there any advantage of using Node instead of Babel Node?